.github/workflows/ci.yml

```diff
@@ -308,11 +308,10 @@ jobs:
     strategy:
       matrix:
         # Focal builds are broken in CI, we can only build bionic for now
-        #suite: [focal, bionic]
-        suite: [bionic]
+        suite: [focal, bionic]
```

Comment:
This is the publish job for the debian packages, which only happens at release time. We have another job at the top of the workflow that builds the debian packages and tests that they install fine.

Reply:
Ah, sorry, my mistake. I've fixed this now